### PR TITLE
Cluster Merging

### PR DESCRIPTION
**Pull request by jtwinrelevanceai**

### CODE REVIEW

**Comment by jtwinrelevanceai** [comment]
PR blocked by missing node endpoint

**Comment by JackyKoh** [comment]
If you fix merge conflict i can approve @jtwinrelevanceai

File: relevanceai/operations/cluster/centroids.py [hunk 1]
```diff
@@ -1,282 +1,282 @@
-from typing import Dict, List, Optional
```
Comment (jtwinrelevanceai) [inline relevanceai/operations/cluster/centroids.py:1]: not sure why its says I deleted this file

**Comment by jtwinrelevanceai** [comment]
@boba-and-beer  I ran `black .` and it blacked the quickstart_guide just whitespace removal

File: relevanceai/_api/endpoints/services/centroids.py [hunk 2]
```diff
@@ -120,7 +80,7 @@ def insert(
             },
         )
 
-    def documents(
+    def list(
```
Comment (boba-and-beer) [inline relevanceai/_api/endpoints/services/centroids.py:83]: You will need to add back a `documents` function. Just add this 1 line: 

```
documents = list
```

Comment (jtwinrelevanceai) [reply relevanceai/_api/endpoints/services/centroids.py:83]: but the end point is called ListCentroids`

Comment (jtwinrelevanceai) [reply relevanceai/_api/endpoints/services/centroids.py:83]: `list` and `documents` methods called the same endpoint anyway. Are you saying you want the function called `documents` instead?

**Review by boba-and-beer — APPROVED** [review]

**Pull request merged**